src/content/release/breaking-changes/default-systemuimode-edge-to-edge.md

```diff
@@ -96,13 +96,19 @@ Modify these default styles in your manifest file:
 </manifest>
 ```
 
-Find where this style is defined in
-`your_app/android/app/src/main/res/values/styles.xml`.
-There, add the following attribute to the style:
+Locate the style definition in:
+`your_app/android/app/src/main/res/values/styles.xml`
+
+Add the following attribute to the appropriate style:
```

Comment:
Since the following snippet showcases updating two different `style` elements:

```suggestion
Add the following attribute to the appropriate styles:
```